ci/gpu/build.sh

```diff
@@ -101,8 +101,8 @@ function install_dask {
     # Install the main version of dask, distributed, and streamz
     gpuci_logger "Install the main version of dask, distributed, and streamz"
     set -x
-    pip install "git+https://github.com/dask/distributed.git@2021.07.1" --upgrade --no-deps
-    pip install "git+https://github.com/dask/dask.git@2021.07.1" --upgrade --no-deps
+    pip install "git+https://github.com/dask/distributed.git@2021.09.1" --upgrade --no-deps
+    pip install "git+https://github.com/dask/dask.git@2021.09.1" --upgrade --no-deps
```

Comment:
Would it be worth grouping these into an environment variable that we set once?

Reply:
Sure, added one

Reply:
Thanks Prem! 😄